app/scripts/views/mixins/password-strength-mixin.js

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'lib/promise',
+  'lib/runtime-require',
+  'lib/url'
+], function (p, runtimeRequire, Url) {
+  'use strict';
+
+  var PasswordStrengthMixin = {
+    initialize: function (options) {
+      this._able = options.able;
```

Comment:
> Cocktail is awesome like that. If a View has the same function as a mixin, it'll create a wrapper around both of them, and when called, call both.

Ah I saw it in action but for some reason didn't make sense here, nevermind. Thanks for the link!